internal/host/statemachine.go

```diff
@@ -637,6 +637,8 @@ func NewHostStateMachine(sm stateswitch.StateMachine, th *transitionHandler) sta
 		If(SufficientOrUnknownInstallationDiskSpeed),
 		If(NonOverlappingSubnets),
 		If(CompatibleAgent),
+		If(NoSkipInstallationDisk),
```

Comment:
These 2 lines are the only "real" change, the rest of the changes are just tests / changes that allow me to test them